### DISCUSSION TITLE
目录中页码数值，如何不加粗

### DISCUSSION THREAD

**Discussion by chiguokun** [zoam/xmu-thesis-grd#27] [Q&A]
现在版本中目录中显示的页码数值是加粗，如何把这个加粗（黑体）去掉。学院给的模板中是没有加粗的，如下图。
![image](https://user-images.githubusercontent.com/15364720/119448635-30522080-bd64-11eb-9d9e-971130051d57.png)

**Comment by zoam** [comment]
c7a3a16

已更新

**Comment by onns** [comment]
这个问题其实可以不用改的。。。

**Reply by zoam** [reply]
不加粗看着顺眼一点就改了。

**Reply by onns** [reply]
hhh好吧

**Comment by insist1995** [comment]
![1abfa83a3ffbc95311fe11a1d1604ca](https://user-images.githubusercontent.com/28755115/119600100-e9723280-be18-11eb-86d3-1fbfd0cf5f82.png)
但是，信息学院的论文模板里，英文目录contents的页码字体大小又是不一样的

**Reply by zoam** [reply]
这个就难倒我了。。。我个人认为统一字体大小比较好看一些。

**Reply by onns** [reply]
问你个问题。。。我们章好像是段前24段后18诶，我今天突然发现段后比段前高？段后是40

**Reply by zoam** [reply]
我也没搞懂。只是目前这么设能达到想要的效果。如果想搞懂还得研究一下。
